src/Antd_Grid.re

```diff
@@ -0,0 +1,165 @@
+open Antd_Helpers;
+
+[%bs.raw {|require("antd/lib/grid/style")|}];
+module Row = {
+  [@bs.module]
+  external reactClass : ReasonReact.reactClass = "antd/lib/grid/row";
+
+  [@bs.deriving jsConverter]
+  type rowType = [ | `flex];
+
+  [@bs.deriving jsConverter]
+  type rowAlign = [ | `top | `middle | `bottom];
+
+  [@bs.deriving jsConverter]
+  type rowJustify = [
+    | `start
+    | [@bs.as "end"] `end_
+    | `center
+    | `spaceAround
+    | `spaceBetween
+  ];
+
+  [@bs.obj] external _gutterBreakpointMap: 
+    (
+      ~xs: string=?,
+      ~sm: string=?,
+      ~md: string=?,
+      ~lg: string=?,
+      ~xl: string=?,
+      ~xxl: string=?,
+      unit
+    ) => _ = "";
+
+  let breakpointGutter = (~xs=?,~sm=?,~md=?,~lg=?,~xl=?,~xxl=?,()) => {
```

Comment:
I don't see where is this getting used in the binding? Are you planning to let user use this function? It would be better if we encoded this into to type system

Reply:
this and complexColSize  I am using like this
```
            <Row gutter=4 align=`top>
                <Col span=12 xs=Col.complexColSize(~span=5,())>  
                    (ReasonReact.stringToElement("A"))
                </Col>
                <Col span=8 xs=3>  
                    (ReasonReact.stringToElement("B"))
                </Col>
            </Row>
            <Row gutter=Row.breakpointGutter(~sm=5, ())>
                <Col span=8>  
                    (ReasonReact.stringToElement("A"))
                </Col>
                <Col span=8>  
                    (ReasonReact.stringToElement("B"))
                </Col>
                <Col span=8>  
                    (ReasonReact.stringToElement("C"))
                </Col>
            </Row>
```

I was trying to work out how to model it with the type system without being too cumbersome.